source/content/guides/wordpress-composer/04-bedrock-composer.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Using Bedrock to Create Your WordPress Pantheon Site
+subtitle: Create a Composer-managed WordPress Site with Bedrock
+description: Learn more about Bedrock for Composer-managed WordPress sites.
+categories: [develop]
+tags: [wordpress]
+contributors: [whitneymeredith]
+layout: guide
+showtoc: true
+permalink: docs/guides/wordpress-composer/bedrock-composer
+anchorid: bedrock-composer
+---
+
+[Composer](https://getcomposer.org/) does not fully support WordPress, however, Bedrock is a central feature of Composer. [Bedrock](https://roots.io/bedrock/) is a custom WordPress project structure that helps simplify working on large projects with multiple plugins and custom code. 
+
+## Create a New Pantheon WordPress Site with Bedrock
+
+1. Run the following command in Terminus:
+
+    ```bash
+    composer create-project your_site
```

Comment:
This command is probably incomplete

Reply:
The Bedrock docs list `composer create-project roots/bedrock` as the start step for installing Bedrock. This command pulls in `roots/bedrock` to be the framework for the Composer-ized project, so replacing `roots/bedrock` with `your_site` would break/not pull in Bedrock.

Reply:
```suggestion
    composer create-project roots/bedrock
```